plugins/providers/virtualbox/cap/validate_disk_ext.rb

```diff
@@ -0,0 +1,27 @@
+require "log4r"
+
+module VagrantPlugins
+  module ProviderVirtualBox
+    module Cap
+      module ValidateDiskExt
+        LOGGER = Log4r::Logger.new("vagrant::plugins::virtualbox::validate_disk_ext")
+
+        # The default set of disk formats that VirtualBox supports
+        DEFAULT_DISK_EXT = ["vdi", "vmdk", "vhd"].freeze
```

Comment:
`.map(&:freeze).freeze` to freeze everything. Otherwise the contents of the array can be modified (`DEFAULT_DISK_EXT.first << 'extra'`)